daemon/log.rs

```diff
@@ -63,3 +65,27 @@ pub fn clean_log_handles(task_id: usize, settings: &Settings) {
         );
     };
 }
+
+/// Return stdout and stderr of a finished process
+/// Everything is compressed using Brotli and then encoded to Base64
```

Comment:
I haven't followed the logic really closely, but is it actually necessary to base64-encode if you're sending it over bincode? Isn't that binary-safe?

Reply:
That was an old comment as well^^
The base64 encode was needed before migrating to bincode, since binary data isn't natively supported in JSON.